R/plumber.R

```diff
@@ -177,10 +182,17 @@ plumber <- R6Class(
       private$notFoundHandler <- default404Handler
 
       if (!is.null(file)){
-        private$lines <- readLines(file)
-        private$parsed <- parse(file, keep.source=TRUE)
+        con <- file(file, encoding=utf8Encoding)
+        on.exit(close(con), add=TRUE)
+
+        private$lines <- readLines(con)
+        Encoding(private$lines) <- utf8Encoding
+
+        srcfile <- srcfilecopy(file, private$lines, file.info(file)[1, "mtime"],
```

Comment:
Any specific reasoning for the `mtime` over `Sys.time()` (when the file is 'copied')?

Reply:
I don't think we need this. Changed.